pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Tutorial-1
```

Comment:
Why is the title "Tutorial-1"? Please fix this line with an appropriate title :)